### PR TITLE
Add get open question set

### LINKED ISSUE #67: Semester repository getQuestionSets
It doesn't need all that complication, it can be done using a transformer for ones that include pivot... @SkaeX @BrainMaestro

### PR DESCRIPTION
**Pull request by BrainMaestro**
Also makes the pivot table info part of the default include in semester transformer.
Fixes #67

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Approved

File: app/Repositories/Contracts/SemesterRepository.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ public function createSemester($season, $year, $currentSemester = false);
 
     public function addQuestionSet($id, $questionSetId, $evaluationType);
 
-    public function getQuestionSets($id);
```
Comment (SkaeX) [inline app/Repositories/Contracts/SemesterRepository.php]: Why are you removing this?

Comment (tenorz007) [inline app/Repositories/Contracts/SemesterRepository.php]: Just leave this MR for now.

**Comment by tenorz007** [comment]
Approved ....for now!

**Pull request merged**